frontend/src/resources/theme.js

```diff
@@ -14,7 +14,8 @@ const color = {
     lightGrayishBlue2: '#DFE0EB',
     paleBlue: '#DDE2FF',
     paleBlueTransparent: 'rgba(221, 226, 255, 0.08)',
-    veryDarkGrayishBlue: '#373a47'
+    veryDarkGrayishBlue: '#373a47',
+    lightgreen: '#b5b4b9',
 };
 
 const typography = {
```

Comment:
This is what we want to do for all our styles.

Maybe we could define Primary and Secondary Colors and use these everywhere. @IremToroslu

Reply:
yes, Let;s group them . I already added more color codes there and saved them as the name of the material type as well. Due to the fact that I have not push it yet, it does not show up in here. Let's also check out the metarial colors and how many materials do we have in general. Then for some of them we can fix the colors on Pie-chart.